tensorboard/plugins/hparams/backend_context_test.py

```diff
@@ -614,6 +614,41 @@ def test_experiment_from_data_provider_session_group(self):
         """
         self.assertProtoEquals(expected_exp, actual_exp)
 
+    def test_experiment_from_data_provider_session_group_without_run_name(self):
+        self._mock_tb_context.data_provider.list_tensors.side_effect = None
+        self._hyperparameters = provider.ListHyperparametersResult(
+            hyperparameters=[],
+            session_groups=[
+                provider.HyperparameterSessionGroup(
+                    root=provider.HyperparameterSessionRun(
+                        experiment_id="exp/session_1", run=""
```

Comment:
Should the id field just be `exp`?

Reply:
No, it has to be the entire path of the session. I left a comment in the test trying to explain for future readers.